module/merintr/inventry.c

```diff
@@ -1062,6 +1076,65 @@ Bool InventoryDropCurrentItem(room_contents_node *container)
    return True;
 }
 /************************************************************************/
+/*
+ * InventoryMoveCurrentItem:  Move the selected item with the inventory cursor 
+ *   to the given (x, y) coordinates, if any.  Return True iff item moved.
+ *   Coordinates (0,0) are at the top left corner of the inventory window,
+ *   increasing as you go to the right and down.
+ */
+Bool InventoryMoveCurrentItem(int x, int y)
+{
+   InvItem *item, *drop_position;
+
+   item = InventoryGetCurrentItem();
+   if (item == NULL)
+      return False;
+
+   // Find row and col in absolute coordinates
+   int row, col;
```

Comment:
Declare where used.

Reply:
This looks ok to me but maybe I am missing it.  I declare int row, col on line 1094 and on 1095 next line we use them.  Is that ok?

Reply:
You should do
```
int row = top_row + y / INVENTORY_BOX_HEIGHT;
```

etc.

Reply:
Thanks, changed how I was doing this.